sidecars/vale/Dockerfile

```diff
@@ -8,7 +8,7 @@
 # Contributors:
 #   Red Hat, Inc. - initial API and implementation
 
-FROM jdkato/vale:v2.8.0 as vale
+FROM jdkato/vale:latest as vale
```

Comment:
It seems odd to use "latest". This will auto-update when a new version is published. Is this what we want?

Reply:
+1 to use tag

Reply:
There is no tag for `v2.10.1` https://hub.docker.com/r/jdkato/vale/tags

Reply:
Issue to crate a tag https://github.com/errata-ai/vale/issues/343